docs/JMAQS_1/utilties/Logger.md

```diff
@@ -1,5 +1,4 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> Logger
-
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Logger
 
 ## Overview
```

Comment:
Multiple references in this file to csharp things.  DLL in copy, code blocks marked as C#.

Reply:
@jason-edstrom  not sure exactly what looks like C# stuff but made some changes. Review needed to confirm if changes are valid.

Reply:
You hit 99% of my concerns for this file.  For the sake of splitting hairs, consider changing `this.log` to `this.getLogger`.  More accurate to the code base. :)